### DISCUSSION TITLE
Welcome to openplaceguide Discussions!

### DISCUSSION THREAD

**Discussion by amenk** [OpenPlaceGuide/openplaceguide#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Na-chidori** [comment]
Hello, My name is Nahom Biniyam and my background is in Electrical and Computer Engineering from Addis Ababa University.
And I am happy to have joined this team.

**Reply by amenk** [reply]
Which programming languages do you know so far? Feel free to clone and setup the project  https://github.com/OpenPlaceGuide/opg-pages locally and ask any questions

**Reply by amenk** [reply]
@Na-chidori how is it going? Do you have any questions? Feel free to open a new discussion and ping me if anything is not clear.

**Comment by Na-chidori** [comment]
Also since I am an intern for this project, I am hoping to learn and do whatever the internship requires of me. And it was recently that I was introduced to POI. So excited to have joined this too.
